Pod/Classes/FBSnapshotTestCase+AGSnapshotHelper.m

```diff
@@ -15,21 +15,47 @@
 
 @implementation FBSnapshotTestCase (AGSnapshotHelper)
 
+- (void)testView:(UIView *)view {
+    [self testView:view contentSizeCategory:UIContentSizeCategoryLarge identifier:nil];
+}
+
 - (void)testView:(UIView *)view contentSizeCategory:(NSString *)contentSizeCategory {
+    [self testView:view contentSizeCategory:contentSizeCategory identifier:nil];
+}
+
+- (void)testView:(UIView *)view contentSizeCategory:(NSString *)contentSizeCategory windowsBoundsSizesMask:(AGWindowBoundsSizesMask)mask {
+
+    NSArray<NSValue *> *sizes  = [AGWindowBoundsSizesMaskConverter windowBoundsSizesForMask:mask];
+    if (sizes == nil) {
 
-    [self testLayer:view.layer contentSizeCategory:contentSizeCategory];
+        [self testView:view contentSizeCategory:contentSizeCategory identifier:nil];
+        return;
+    }
+
+    for (NSValue *sizeValue in sizes) {
+
+        CGSize size = sizeValue.CGSizeValue;
+        CGRect newFrame = view.frame;
+        newFrame.size = size;
+        view.frame = newFrame;
+        [view layoutIfNeeded];
+
+        NSString *identifier = [AGWindowBoundsSizesMaskConverter nameForSize:size];
+
+        [self testView:view contentSizeCategory:contentSizeCategory identifier:identifier];
+    }
 }
 
-- (void)testLayer:(CALayer *)layer contentSizeCategory:(NSString *)contentSizeCategory {
```

Comment:
CALayer is lower layer in view hierarchy which. `autoresizingMask` and `NSLayoutConstraint` works on UIViews instances so there's not point to test na CALayers in this repository.